### PR TITLE
4.13 branch cut changes for cnf-features-deploy.

### PR DESCRIPTION
**Pull request by nishant-parekh**
configuration changes
   jobs update
   image mapping update

### CODE REVIEW

File: ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-release-4.13.yaml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    namespace: ocp
+    tag: base
+  openshift_release_golang-1.19:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+  os-min8:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
```
Comment (cgoncalves) [inline ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-release-4.13.yaml:13]: change base image to ubi9

```suggestion
  os-min9:
    name: ubi-minimal
    namespace: ocp
    tag: "9"
```

File: ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-release-4.13.yaml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.19
+  to: cnf-tests
+- context_dir: tools/s2i-dpdk/
+  dockerfile_path: Dockerfile
+  from: base
+  to: dpdk
+- context_dir: .
+  dockerfile_path: ztp/resource-generator/Containerfile
+  from: os-min8
```
Comment (cgoncalves) [inline ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-release-4.13.yaml:33]: change base image to ubi9
```suggestion
  from: os-min9
```

File: ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-release-4.13.yaml (hunk 1 above)
Comment (nishant-parekh) [reply ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-release-4.13.yaml:13]: I am not sure if 4.13 will ship with rhel-9 still as the testing is on going.. so I didnt make the update in this change.. once we have agreement this change can follow the branch cut.

File: ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-release-4.13.yaml (hunk 2 above)
Comment (nishant-parekh) [reply ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-release-4.13.yaml:33]: same as above

File: ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 base_images:
   base:
-    name: "4.13"
+    name: "4.14"
```
[the remark this answers is not in the document]
Comment (nishant-parekh) [reply ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml:3]: same as above

File: ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-release-4.13.yaml (hunk 1 above)
Comment (lack) [reply ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-release-4.13.yaml:13]: I think that's still the plan of record, but it's true we're not actually there yet :(

**Review by ffromani — COMMENTED** [review]
/lgtm

**Comment by lack** [comment]
/pj-rehearse skip
We know the gcp tests are flaky already.

**Pull request merged**